### ISSUE TITLE
Type definations not working

### ISSUE THREAD

**Issue by yash-dropbase** [quicktype/glide-data-grid#752]
Hey thanks for the project!

We are using vite + react-ts on our client and the package cannot resolve types from the package. Seeing below error: 

```
Could not find a declaration file for module '@glideapps/glide-data-grid'. '.../node_modules/@glideapps/glide-data-grid/dist/js/index.js' implicitly has an 'any' type.
  There are types at '.../node_modules/@glideapps/glide-data-grid/dist/ts/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@glideapps/glide-data-grid' library may need to update its package.json or typings.
```

I can confirm this fix: https://github.com/quicktype/glide-data-grid/pull/747 works for me.

Only solution would be upgrading the package version in order to include the types export, let me know if there's any solution I can do on the user's code to include types.

Thanks!

**Comment by yash-dropbase** [comment]
@jassmith any ETA when the new version will be up ?

**Comment by chkn** [comment]
@yash-dropbase We just published [5.2.2-beta5](https://www.npmjs.com/package/@glideapps/glide-data-grid/v/5.2.2-beta5) a couple days ago, which should include #747. Closing this as the fix is released.

**Comment by yash-dropbase** [comment]
Thanks @chkn :D Really appreciate!
